2020/8xxx/CVE-2020-8557.json

```
@@ -1,18 +1,148 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "security@kubernetes.io",
+        "DATE_PUBLIC": "2020-07-15T00:00:00.000Z",
         "ID": "CVE-2020-8557",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "Kubernetes node disk Denial of Service by writing to container /etc/hosts"
     },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Kubernetes",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "prior to 1.18.6"
```

Consider using the machine readable format?
```suggestion
                                            "version_affected": "<",
                                            "version_name": "1.18",
                                            "version_value": "1.18.6"
```